build_binaries.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 set -ex
 cd Surelog && make PREFIX=$PWD/../image/ release install -j $(nproc) && cd ..
-autoconf && ./configure --prefix=$PWD/image && make -j $(nproc) && make install
+autoconf && ./configure --prefix=$PWD/image SURELOG_INSTALL_DIR=$PWD/image && make -j $(nproc) && make install
```

Comment:
How about we just set `SURELOG_INSTALL_DIR` at the beginning and then feed it to Surelog's `PREFIX`?

Reply:
Done